### DISCUSSION TITLE
Reembolso.

### DISCUSSION THREAD

**Discussion by ElainePrazeres** [aclap-dev/video-downloadhelper#1092] [License & Purchase requests]
### What's your issue?

Não Funciona o produto. Quero reembolso. Já acionei os meus direitos do consumidor. Quero o reembolso imediato.
![Captura de tela 2024-08-03 210052](https://github.com/user-attachments/assets/7e0587e5-3cbc-4afb-ac5d-e072ea989247)
![Captura de tela 2024-08-03 210052](https://github.com/user-attachments/assets/d35df4c8-f7a3-483c-b43f-f32bb6863630)

### The 10 first digits (only 10) of your license number or email address

acc27e0424

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
You have been refunded. Should be available in a few business days.
